packages/containers/Import/Steps/ImportingStep/ImportingStep.js

```diff
@@ -64,16 +58,9 @@ const ImportingStep = ({ progress, onClickToDashboard, onClickToStart }) => (
           Don’t close the browser because migration process will stop!
         </WarningText>
       </TwoItemsWrapper>
-      <StyledButton
-        color="white"
-        disabled={progress < PROGRESS_THRESHOLD}
-        onClick={onClickToStart}
-      >
-        IMPORT ONE MORE *.CSV
-      </StyledButton>      
       <Button
         color="white"
-        disabled={progress < PROGRESS_THRESHOLD}
+        disabled={progress < IMPORT_PROGRESS_THRESHOLD}
         onClick={onClickToDashboard}
       >
         GO TO DASHBOARD
```

Comment:
I know, this is not your code, but could you pls change to `Go to dashboard` too?

Reply:
Done